manager/src/main/resources/application.properties

```diff
@@ -40,6 +40,10 @@ event-bridge.kafka.client.secret=${EVENT_BRIDGE_KAFKA_CLIENT_SECRET}
 event-bridge.kafka.security.protocol=${EVENT_BRIDGE_KAFKA_SECURITY_PROTOCOL:SASL_SSL}
 event-bridge.kafka.topic-prefix=ob-
 
+# ManagedResource provisioning worker
+event-bridge.resources.worker.max-retries=3
```

Comment:
Can you make it possible for us to override by environment variables if we want to.

Reply:
Added in new commit. 

I believe it to be the correct way; but documentation seems a bit sparse (Quarkys, SmallRye, MicroProfile)